ocs_ci/deployment/vmware.py

```diff
@@ -751,3 +705,81 @@ def clone_openshift_installer():
             constants.VSPHERE_INSTALLER_REPO, upi_repo_path,
             f'release-{ocp_version}'
         )
+
+
+def change_mem_and_cpu():
```

Comment:
Do you think it's worth to have it parametrized with params like worker_num_cpus, master_num_cpus, .. ?

By default set them to None and if not specified you will just load default value from config as you do below? If so we can do the same for other functions if you think it can be useful.

Reply:
Ack. I will take it up in another PR.